src/web/components/SignInGate/gateDesigns/copy-opt-test/SignInGateCopyOptVar.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { cx } from 'emotion';
+
+import { LinkButton } from '@guardian/src-button';
+import { Link } from '@guardian/src-link';
+import { trackLink } from '@frontend/web/components/SignInGate/componentEventTracking';
+import { SignInGateProps } from '../types';
+import {
+	actionButtons,
+	bodyBold,
+	bodyText,
+	faq,
+	firstParagraphOverlay,
+	headingStyles,
+	hideElementsCss,
+	laterButton,
+	registerButton,
+	signInGateContainer,
+	signInHeader,
+	signInLink,
+} from '../shared';
+
+export const SignInGateCopyOptVar = ({
+	signInUrl,
+	guUrl,
+	dismissGate,
+	abTest,
+	ophanComponentId,
+	isComment,
+	signInGateCopy,
+}: SignInGateProps) => {
+	return (
+		<div
+			className={signInGateContainer}
+			data-cy={`sign-in-gate-${abTest?.variant}`}
+		>
+			<style>{hideElementsCss}</style>
+			<div className={firstParagraphOverlay(!!isComment)} />
+			<h1 className={headingStyles}>{signInGateCopy?.header}</h1>
+			<p className={bodyBold}>{signInGateCopy?.subHeader}</p>
+			{signInGateCopy?.paragraphs.map((paragraph) => {
+				return <p className={bodyText}>{paragraph}</p>;
```

Comment:
Looking at `SignInGateMain.tsx ` the text `privacy settings` is a button.  Did you mean to change this in the copy test?

Reply:
I have added a fix for this if you wouldn't mind re-reviewing?